src/main/scala/zio/rocksdb/TransactionDB.scala

```diff
@@ -0,0 +1,53 @@
+package zio.rocksdb
+
+import org.{ rocksdb => jrocks }
+import zio._
+
+object TransactionDB extends Operations[TransactionDB, service.TransactionDB] {
+  private final class Live(db: jrocks.TransactionDB) extends RocksDB.Live(db, Nil) with service.TransactionDB {
```

Comment:
The `Nil` passed to the `RocksDB.Live` constructor doesn't look right. Are we saying that `TransactionDB` only supports the default column family?

Reply:
No. I think we can add that capability later. I though having transactional capabilities first and stabilizing the API right now would be more important than increase it's surface area. 

There are multiple other variants for other functions that are available but I have refrained my self from adding them either.

Reply:
Sounds good!